app/views/posts/index.html.erb

```diff
@@ -1,7 +1,10 @@
 <% @posts.order(created_at: :desc).each do |post| %>
+
   <div id="post">
     <div id="post-body">
-      <%= image_tag post.image_url.to_s %>
+      <% if post.image_url %>
```

Comment:
This seems to display exactly the same way if I comment out lines 5 and 7 (I tried this with a mixture of posts that had only text and posts that had only images).

I suspect that line 6 already only runs if post.image_url is true, and that therefore the 'if' statement is redundant...